lib/omniauth/strategies/google_oauth2.rb

```diff
@@ -46,8 +46,8 @@ def authorize_params
       info do
         prune!(
           name: raw_info['name'],
-          email: raw_info['email'],
-          email_verified: raw_info['email_verified'],
```

Comment:
NIT: Lets keep `email_verified` as a key, that way we don't break people that used this field in 0.6.1.

Reply:
:+1: done